scripts/run_docker_build.sh

```diff
@@ -40,7 +40,7 @@ conda install --yes anaconda-client obvious-ci
 conda install --yes conda-build-all
 
 conda info
-unset LANG
+export LANG=en_US.UTF-8
```

Comment:
This has changed the global staged-recipes setup, not just that for runipy. I don't know if that is healthy or not, but I must have put the unset there for a reason... either way, we need a comment adding to this. @jakirkham - would you mind opening a PR which adds such a comment?